.travis.yml

```diff
@@ -1,11 +1,23 @@
 language: go
+sudo: required
 
 go:
     - 1.6
     - tip
 
 go_import_path: github.com/01org/ciao
 
+install:
+   - go install -v github.com/01org/ciao/ssntp/ciao-cert
```

Comment:
I think if we used before_script instead of install we wouldn't need to build ciao-cert explicitly.  travis is doing a go get ./... during install so it should build ciao-cert with everything else.  Just conjecture though.  We'd need to try to see.